README.md

```diff
@@ -73,7 +73,7 @@ free and open source deep learning models.
 To run the docker image, which automatically starts the model serving API, run:
 
 ```
-$ docker run -it -p 5000:5000 codait/max-object-detector
+$ docker run -it -p 5000:5000 quay.io/codait/max-object-detector
```

Comment:
Did you mean to include this, if so why?

Reply:
Yes. We moved from dockerhub to quay a month or 2 ago.

Reply:
Oh ok, then you should update the reference to Docker Hub in the line below